routes/theme_routes.py

```
@@ -0,0 +1,188 @@
+            settings_service=registry.settings_service,
+        )
+
+    @theme_bp.route("", methods=["GET"])
+    def list_themes():
+        """获取已安装主题列表。"""
+        try:
+            themes = _theme_service().list_themes()
+            active = _theme_service().get_active_theme()
+        except ThemeError as e:
+            return jsonify({"success": False, "message": str(e)}), 400
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)